src/main/java/com/williamfeliciano/sba_movies/dtos/Rating.java

```diff
@@ -0,0 +1,19 @@
+package com.williamfeliciano.sba_movies.dtos;
+
+public enum Rating {
```

Comment:
What's the advantage of having an enum which has its values named by an index? Using a an integer wouldn't be easier?